src/sources/rlcsv3/rlcs.py

```
@@ -66,30 +68,33 @@ def parse_rlcs(line: str | bytes) -> dict[str, str | Number] | None:
-        res.update({"Heater Resistance 2": (res["Heater Kelvin High 2 Voltage"] - res["Heater Kelvin Low 2 Voltage"])/res["Heater Current 2"]})
+        res["Heater Resistance 2"] = (res["Heater Kelvin High 2 Voltage"] - res["Heater Kelvin Low 2 Voltage"])/res["Heater Current 2"]
     else:
-        res.update({"Heater Resistance 2": 0.0})    
+        res["Heater Resistance 2"] = 0.0
 
-    return res
-        
- 
+    try:
+        return RLCSv3Message(id=0, data=res, message_version=2)
+    except Exception as e:
```

Ruff (0.14.5)

Do not catch blind exception: `Exception`

(BLE001)